grpc/src/main/java/com/replica/RouterImpl.java

```diff
@@ -167,6 +174,104 @@ public void routeStreetMode(StreetRouteRequest request, StreamObserver<StreetRou
         }
     }
 
+    @Override
+    public void routeCustom(CustomRouteRequest request, StreamObserver<StreetRouteReply> responseObserver) {
```

Comment:
This endpoint is only used by the sandbox; standard routers will be built using the YAML-specified custom models we provide, but then will just use the normal `routeStreetMode` endpoint to get routes